### PR TITLE
Improved test

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-1026

### Change description ###
Minor improvement in test class

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by lgonczar** [comment]
Nice. Now that we're creating the task from tests, it would be good to move `FROM` and `recipients` global fields to the method(s) where they're used.

**Comment by apereverzin** [comment]
FROM and recipients are used in several tests and in captors so it makes sense to define them as class attributes not in methods

**Comment by lgonczar** [comment]
> FROM and recipients are used in several tests and in captors so it makes sense to define them as class attributes not in methods

They're only used at test level, so they can definitely be moved to tests.

File: src/test/java/uk/gov/hmcts/reform/blobrouter/tasks/SendDailyReportTaskTest.java [hunk 1]
```diff
@@ -113,6 +103,12 @@ void sendReport_should_call_email_sender() throws Exception {
     @Test
     void sendReport_should_not_call_email_sender_if_csv_writer_throws() throws Exception {
         // given
+        String from = "From";
+
+        String[] recipients = new String[]{"rec1", "rec2"};
+
+        sendDailyReportTask = getSendDailyReportTask(from, recipients);
```
Comment (lgonczar) [inline src/test/java/uk/gov/hmcts/reform/blobrouter/tasks/SendDailyReportTaskTest.java]: These two can even just be passed as values, but that's only a tiny suggestion.

**Pull request merged**